### PR TITLE
feature/ECCW-655-style-event-cards

### PR DESCRIPTION
**Pull request by kurtisRogers-Nomensa**
### What have I done?
- Added config for card display mode
- Added view block for event cards
- Added styles so that they're identical to the news cards

### CODE REVIEW

**Comment by kurtisRogers-Nomensa** [comment]
image

File: config/default/cludo_search.settings.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 customerId: '2222'
-engineId: '10581'
```
Comment (Polynya) [inline config/default/cludo_search.settings.yml:2]: Revert this change

File: config/default/system.site.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ slogan: ''
 page:
   403: /node/1212
   404: /node/1212
-  front: /node/1211
```
Comment (Polynya) [inline config/default/system.site.yml:11]: Revert this change

File: config/default/webform.webform.feedback_form.yml [hunk 3]
```diff
@@ -13,14 +13,6 @@ title: 'Feedback Form'
 description: "For providing feedback to content\r\n<div id=\"extension-host\">&nbsp;</div>"
 category: ''
 elements: |-
-  info:
```
Comment (Polynya) [inline config/default/webform.webform.feedback_form.yml:16]: Revert this file

**Review by Polynya — CHANGES_REQUESTED** [review]
It's best to `git checkout` any config changes you know aren't related to the ticket. I can do revert the files before this is merged.

**Comment by MatthewWilkes** [comment]
@kurtisRogers-Nomensa Can you also change the title of this PR to explain what it does and look at combining the description and your first comment into something more aligned with the PR template we discussed last month?

**Pull request closed without merging**